### ISSUE TITLE
Error while deleting rules with --docker

### ISSUE THREAD

**Issue by rukmarr** [thombashi/tcconfig#108]
I'm using tcconfig to manage docker container network interfaces. Every time I'm trying to delete previously set rules with --docker, I'm getting this error:
```
[ERROR] tcconfig: command execution failed
  command=/usr/sbin/tc qdisc del dev veth7f0e1f0 ingress
  stderr=Error: Cannot find specified qdisc on specified device.

[WARNING] tcconfig: [Errno 2] Bad argument `iptables'
Try `iptables -h' or 'iptables --help' for more information.
 (can not delete iptables entries)
```
The rules, however, cease to affect container.

How to reproduce:
```
sudo docker run -d alpine --name test_ping ping 8.8.8.8
sudo  tcset --delay 100ms --docker test_ping
sudo tcdel --docker test_ping
```

**Comment by thombashi** [comment]
@rukmarr 
Thank you for your report.

Unfortunately, I could not reproduce the error with my environment (Ubuntu 18.04, Python 2.7/3.7, iproute2 4.15.0-2ubuntu1).
Could you tell me your environments?

**Comment by rukmarr** [comment]
@thombashi 
Sure, I'm using Fedora 29, Python 2.7.15/3.7.0, iproute2 4.20.0-1.fc29

**Comment by thombashi** [comment]
Thank you for your information.

I will try to reproduce the issue.
This may take some time.

**Comment by thombashi** [comment]
@rukmarr
I had fixed the issue at `tcconfig 0.21.9`.
Please upgrade `tcconfig` and re-execute.

I can reproduce the error at Fedora.
Looks like the error will occur at Fedora(RHEL) environments.

**Comment by thombashi** [comment]
I'll close the issue.
Feel free to reopen if you still have any problems about the issue.
